src/cpx_constrs.jl

```diff
@@ -349,8 +349,10 @@ function add_sos!(model::Model, sostype::Symbol, idx::Vector{Int}, weight::Vecto
 end
 
 add_indicator_constraint(model::Model, idx, coeff, sense, rhs, indicator) =
+    add_indicator_constraint(model::Model, idx, coeff, sense, rhs, indicator, 0) =    
```

Comment:
lol yeah i just realized that -_- sorry

On Fri, Feb 12, 2016 at 5:14 PM, Joey Huchette <notifications@github.com>
wrote:

> In src/cpx_constrs.jl
> <https://github.com/JuliaOpt/CPLEX.jl/pull/63#discussion_r52805803>:
>
> > @@ -349,8 +349,10 @@ function add_sos!(model::Model, sostype::Symbol, idx::Vector{Int}, weight::Vecto
> >  end
> >
> >  add_indicator_constraint(model::Model, idx, coeff, sense, rhs, indicator) =
> > +    add_indicator_constraint(model::Model, idx, coeff, sense, rhs, indicator, 0) =
>
> I think the = at the end of this line breaks things
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/JuliaOpt/CPLEX.jl/pull/63/files#r52805803>.
>